changing-digital-climate.html

```diff
@@ -0,0 +1,734 @@
+---
+layout: default
+title: "Part 3 - Changing the Digital Climate"
```

Comment:
Could we update this to follow the yaml frontmatter in the other sections?

Reply:
Just gonna bump on this @shaqsingh as it'll be a requirement to have elements render correctly

Reply:
sorry-- not super clear on how it differs currently